stories/Display/TabsStory.tsx

```diff
@@ -134,6 +144,196 @@ const TabsStory = () => (
         <Tabs.Pane tab="Salary">Rp 10,000,000</Tabs.Pane>
       </Tabs>
     </StorybookComponent>
+
+    <Divider theme="grey" />
+
+    <StorybookComponent
+      title="Tabs"
+      code="import { Tabs } from 'glints-aries'"
+      usage={`<Tabs variant="underlined" colour="grey">
+        <Tabs.Pane tab="Job">
+        Software Engineer <Badge label="1" />
+        </Tabs.Pane>
+        <Tabs.Pane tab="Company">Glints</Tabs.Pane>
+        <Tabs.Pane tab="Location">Jakarta</Tabs.Pane>
+        <Tabs.Pane tab="Salary">Rp 10,000,000</Tabs.Pane>
+      </Tabs>`}
+    >
+      <Heading style={{ fontSize: '20px', marginBottom: '1em' }}>
+        Horizontal Tabs (Grey Colour, Underlined Variant)
+      </Heading>
+      <Tabs
+        variant={EHorizontalTabVariant.UNDERLINED}
+        colour={ETabColourVariant.GREY}
+      >
+        <Tabs.Pane tab="Job">
+          Software Engineer <Badge label="1" />
+        </Tabs.Pane>
+        <Tabs.Pane tab="Company">Glints</Tabs.Pane>
+        <Tabs.Pane tab="Location">Jakarta</Tabs.Pane>
+        <Tabs.Pane tab="Salary">Rp 10,000,000</Tabs.Pane>
+      </Tabs>
+    </StorybookComponent>
+
+    <Divider theme="grey" />
+
+    <StorybookComponent
+      title="Tabs"
+      code="import { Tabs } from 'glints-aries'"
+      usage={`<Tabs variant="underlined" colour="grey">
+        <Tabs.Pane tab={<div><FileAlternateIcon />Jobs</div>}>
+        Software Engineer <Badge label="1" />
+        </Tabs.Pane>
+        <Tabs.Pane tab={<div><FileAlternateIcon />Company</div>}>Glints</Tabs.Pane>
```

Comment:
I think we can have an `icon` prop instead of makeing `tab` props accept `React.ReactNode` like material-ui https://material-ui.com/zh/components/tabs/#icon-tabs

Reply:
https://github.com/glints-dev/glints-aries/issues/208#issuecomment-647971997 it was requested here to make it accept elements too. Also this was a limitation, that formatted message or <Style>tab heading</Styled> couldn't be added here, resulting in css change through classNames.
Therefore I didn't see a point of adding an additional prop?